tox.ini

```diff
@@ -15,7 +15,7 @@ commands =
   python -m testtools.run {posargs}
 
 whitelist_externals = bash
-passenv = PYTHONPATH
+passenv =
```

Comment:
passenv should be set to PYTHONPATH.
tox had issues on William's blade where we had to set this.

Reply:
Okay